modules/core/src/parallel.cpp

```diff
@@ -494,9 +519,17 @@ int cv::getNumThreads(void)
 
 #if defined HAVE_TBB
 
+#if TBB_INTERFACE_VERSION >= 9100
+    return tbbArena.max_concurrency();
+#elif TBB_INTERFACE_VERSION >= 8000
+    return numThreads > 0
+        ? numThreads
+        : tbb::task_scheduler_init::default_num_threads();
```

Comment:
Not sure what to do here as max_concurrency is only added in TBB2017 (which is 9100 I believe)
https://software.intel.com/en-us/articles/intel-threading-building-blocks-release-notes#2017